### PR TITLE
Update with more detailed and universal instructions for debian-like instances on Huawei and ZTE modems

### PR DESCRIPTION
**Pull request by tiagofreire-pt**
## Proposed change
Add a more universal recipe to switch the operational mode into "modem only" for Huawei, ZTE and similar devices, not exclusively on Raspberry PI but on debian-like distros. Such as Proxmox-VE.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Added the known working modem `ZTE K3565-Z`.

- Link to parent pull request in the codebase: N/A
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sms.markdown [hunk 1]
```diff
@@ -97,13 +98,17 @@ Need to unlock it using [this guide](http://blog.asiantuntijakaveri.fi/2015/07/c
...
-For some unknown reason, the rule that converts these modems from storage devices into serial devices does not run automatically. To work around this problem, follow the procedure to create `udev` rule on a configuration USB stick for the device to switch to serial mode.
+For some unknown reason, the rule that converts these modems from storage devices into serial devices may not run automatically. To work around this problem, follow the procedure below to change the modem mode and (optionally) create `udev` rule on a configuration USB stick for the device to switch to serial mode persistently.
 
-0. Try disable virtual cd-rom and change work mode "only modem". After this modem correct work on Raspberry Pi without 'udev' rule.
+1. Install the `usb_modeswitch` software to switch the modem operational mode (for Debian/Ubuntu distros):
 
-1. Run `lsusb`, its output looks like this:
+```bash
+sudo apt update && sudo apt install usb-modeswitch -y
+```
```
Comment (frenck) [inline source/_integrations/sms.markdown]: This will not work for users of Home Assistant OS, Container or Supervised.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sms.markdown (hunk 1 above)
Comment (tiagofreire-pt) [reply source/_integrations/sms.markdown]: Just added that indication on the instructions. Thanks.

File: source/_integrations/sms.markdown [hunk 2]
```diff
@@ -97,13 +98,17 @@ Need to unlock it using [this guide](http://blog.asiantuntijakaveri.fi/2015/07/c
 
 Search in the [Gammu database](https://wammu.eu/phones/) for modems with AT connection.
 
-### Huawei modems on Raspberry Pi (and similar) devices
+### Huawei/ZTE modems (and similar) devices - NOT applicable for users of Home Assistant OS, Container or Supervised.
```
Comment (frenck) [inline source/_integrations/sms.markdown:84]: That is not correct. The previous instructions did work, now it doesn't apply anymore.
This PR renders the documentation incorrect for our largest user group (while previously it was correct).

Comment (tiagofreire-pt) [reply source/_integrations/sms.markdown:84]: I'm a user of a debian instalation and the problem persists as `Bullseye` release, not being exclusive of the debian distro (raspbian) for the Raspberry PIs. The issue seems to exist in all debian-like distros for some unknown reason.

This is a workaround to enable a proper setup of this USB modems made by ZTE/Huawei (and probably many others). As long they have mass storage functionality within the modem...

**Review by frenck — APPROVED** [review]
Thanks, @tiagofreire-pt 👍

**Pull request merged**